services-manager/src/main/resources/application-services.yaml

```diff
@@ -19,64 +19,37 @@
 # Extract the project version (x.y.z-REVISION etc.)
 app:
   version: '@project.version@'
-# While the library happens to be the same version as services, it should be declared here as a static name
```

Comment:
Since the default use of the manager is for running services for the example, why add the example profile?

Reply:
Reasonable to expect either method